src/test/script_standard_tests.cpp

```diff
@@ -216,7 +216,7 @@ BOOST_AUTO_TEST_CASE(script_standard_ExtractDestination)
     s << OP_0 << ToByteVector(pubkey.GetID());
     BOOST_CHECK(ExtractDestination(s, address));
     WitnessV0KeyHash keyhash;
-    CHash160().Write(pubkey).Finalize(keyhash.begin());
+    CHash160().Write(pubkey).Finalize(keyhash);
```

Comment:
In commit "Make CHash256/CHash160 output to Span" (b92907e0b89210ecf25baa70702550f643d53525)

This commit doesn't seem to compile. Also not sure reason scriptHash.begin() is changing to scriptHash.data() above on line 102.

Reply:
Fixed.